crates/js-sys/src/lib.rs

```diff
@@ -1391,10 +1391,10 @@ impl BigInt {
     /// Returns a tuple of this [`BigInt`]'s absolute value along with a
     /// [`bool`] indicating whether the [`BigInt`] was negative.
     fn abs(&self) -> (Self, bool) {
-        if self >= &BigInt::from(0) {
-            (self.clone(), false)
-        } else {
+        if self < &BigInt::from(0) {
             (-self, true)
+        } else {
+            (self.clone(), false)
```

Comment:
I swapped these to make it more readable. Now if the condition is `true`, then the bool is `true`.